modules/nvidia_plugin/src/cuda_plugin.cpp

```diff
@@ -215,7 +215,9 @@ ov::Any Plugin::get_property(const std::string& name, const ov::AnyMap& properti
     } else if (METRIC_KEY(SUPPORTED_CONFIG_KEYS) == name) {
```

Comment:
Do we need migrate legacy API to new API in this PR as well? @ilya-lavrenov

Reply:
NVidia plugin is fully on API2.0, these metrics are for compatibility purpose